about-peer-review/pyopensci-related-joss-ropensci.md

```diff
@@ -0,0 +1,47 @@
+# How Are JOSS, rOpenSci and pyOpenSci Related? 
+
+## How is pyOpenSci different from JOSS and other review processes?
+
+pyOpenSci aligns closely with the broad mission of JOSS to provide maintainers with credit for their efforts in developing the open source
+tools necessary to support reproducible open science workflows. However, 
+we are not a typical publisher or Journal. Rather we are a community 
+that provides support for both a diverse group of software maintainers and 
+long term maintenance of our packages.
+
+### Why Two Review Processes JOSS and pyOpenSci? 
+
+The pyOpenSci review process is different from that of JOSS in a few ways:
+
+1. Our review is specifically design to enforce modern, community-accepted best practices for Python packaging. 
+1. pyOpenSci places heavy emphasis on documentation and usability in our reviews and associated standardization of both.
+1. pyOpenSci builds community around and visibility for its tools.
+1. We will promote packages and package maintainers once they are accepted into our ecosystem.  
+1. We support long term maintenance of packages. If the maintainer needs to step down, we will ensure a new maintainer takes over OR sunset and remove the package from our ecosystem.
+1. We provide a welcoming forum to ask questions and get help with maintaining your package as needed.  
+
+JOSS reviews are  also [more limited in scope](https://joss.readthedocs.io/en/latest/review_criteria.html) compared to pyOpenSci and the
+[submission criteria](https://joss.readthedocs.io/en/latest/review_criteria.html)
+are, in places, less stringent than those of pyOpenSci.
+
+### JOSS is a Journal and a Publisher 
+
+The JOSS review process is about publication. A review from JOSS will 
+provide you with a citable DOI that is cross-ref
+enabled.
+
+## You can publish in both JOSS and be reviewed by pyOpenSci 
+
+You don't have to chose between pyOpenSci and JOSS; You can submit your package to both.
+
+pyOpenSci and [the Journal of Open Source Software (JOSS)](https://joss.theoj.org/)
+are complementary, partner organizations; and you don't have to chose one or the 
+other! After a package to pyOpenSci has been reviewed and accepted by pyOpenSci
+you can chose to also register it with JOSS. JOSS has [more limited scope](https://joss.readthedocs.io/en/latest/review_criteria.html)  of the 
+for packages that it will review. For instance while pyOpenSci will review and 
```

Comment:
```suggestion
you can also choose to register it with JOSS. JOSS has [more limited scope](https://joss.readthedocs.io/en/latest/review_criteria.html) for packages that it will review. For instance, while pyOpenSci will review and 
```